packages/apps/fortune/exchange-oracle/server/src/common/constant/index.ts

```diff
@@ -1 +1,2 @@
 export const HEADER_SIGNATURE_KEY = 'human-signature';
+export const ESCROW_FAILED_ENDPOINT = '/fortune/escrow-failed-webhook';
```

Comment:
I think this endpoint is worng
```suggestion
export const ESCROW_FAILED_ENDPOINT = '/job/escrow-failed-webhook';
```